cdap-watchdog/src/main/java/co/cask/cdap/logging/context/BatchAdapterLoggingContext.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright © 2015 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.logging.context;
+
+/**
+ * Logging context for instances of ETL templates
```

Comment:
The initial idea I had was to have two different logging contexts. However, after exploring a bit more, removed the necessity of a new logging context. We just have an extra, optional tag in ``ApplicationLoggingContext`` now.